snuba/datasets/transactions.py

```diff
@@ -102,4 +103,5 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             BasicFunctionsProcessor(),
             ApdexProcessor(),
             ImpactProcessor(),
+            TimeSeriesColumnProcessor(self._TimeSeriesDataset__time_group_columns),
```

Comment:
please don't access a private attribute of the parent class (yes I know there are no real private attributes in python but the `__` is meant for you not to access it).  You have this field accessible in the constructor of this class.